### PR TITLE
Add missing entry to changelog

### PR DESCRIPTION
**Pull request by cmelchior**
@realm/java

### CODE REVIEW

**Comment by dalinaum** [comment]
:+1:

**Comment by dalinaum** [comment]
retest this please

**Comment by kneth** [comment]
:+1:

**Comment by stk1m1** [comment]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,12 @@
 * RealmCollection and OrderedRealmCollection have been added. RealmList and RealmResults both implement these interfaces.
 * RealmBaseAdapter now accept an OrderedRealmCollection instead of only RealmResults.
 * RealmObjectSchema.isPrimaryKey(String) (#2440)
+* RealmConfiguration.initialData() can now be used to populate a Realm file before it is used for the first time.
+
+### Credits
```
Comment (emanuelez) [inline CHANGELOG.md]: Maybe credits can be a subsection? With an extra `#`

Comment (cmelchior) [inline CHANGELOG.md]: Tried it. Looks a bit strange, so I think it is better to keep the current nesting. `###` looks really big though so we should probably consider switching all subsections to `####` to make it a bit easier on the eye.

**Pull request merged**